projects/pipelines/testing/test.py

```diff
@@ -211,6 +212,14 @@ def prepare_cluster():
         parallel.delayed(prepare_sutest_scale_up)
         parallel.delayed(prepare_rhods)
 
+    # Update the MAX_CONCURRENT_RECONCILES
+    max_concurrent_reconciles = config.ci_artifacts.get_config("tests.pipelines.max_concurrent_reconciles")
+    updated_param = {
+        "data": {
+            "MAX_CONCURRENT_RECONCILES": str(max_concurrent_reconciles)
+        }
+    }
+    run.run(f"oc patch cm data-science-pipelines-operator-dspo-parameters -n redhat-ods-applications --patch '{json.dumps(updated_param)}'")
```

Comment:
@drewrip , are you sure that this config map isn't reconcilated to its default value by the RHODS operator?
(just asking, as it's often the case)

Reply:
I didn't notice that in my small scale test, but maybe I didn't wait long enough for it to be reconciled. I'm not sure though, this didn't cross my mind.

Reply:
I wanted to check in the CI `gather-extra` ... but it wasn't captured 😅 
just double check after >10min. If your config is still there, you're good, they took care of it.
In general, as soon as you modify something that was deploy by an operator (=not a resource you created yourself), you need to make sure that the operator owning the CRD doesn't reconciliate it. 
Either the operator allows the modification, because the devs tooks care of it (should be the case here), or you need to turn off the operator so that it doesn't overwrite your update

Reply:
Gotcha! That's a good point. I'll launch something longer to make sure. Hopefully they'll allow the edit. I wasn't able to figure out how I could change these defaults while deploying the operator itself and this seemed simpler but we'll see if it works :)